app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,64 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[ show edit update destroy ]
+
+  def index
+    @foods = Food.all
+  end
+
+  def show; end
+
+  def new
+    @food = Food.new
+  end
+
+  def edit; end
+
+  def create
+    @food = Food.new(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to @food, notice: "Food was successfully created." }
+        format.json { render :show, status: :created, location: @food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @food.update(food_params)
+        format.html { redirect_to @food, notice: "Food was successfully updated." }
+        format.json { render :show, status: :ok, location: @food }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
```

Comment:
interesting, did not know about status with render. If you are sure that it works then ok 👍